### DISCUSSION TITLE
How to get VSCode intellisense/autocomplete for expect

### DISCUSSION THREAD

**Discussion by cyberwombat** [vitest-dev/vitest#2730] [General]
First I had the same issue when using Jest. I have a no intellisense for the expect functions. I installed @types/jest which seems to be the solution for some but that never fixed issue and the same issue is now in vitest. 

For example - `expect(true).toBeDefined()` - I would want VSCode to offer all the "toBe..." suggestions when I start typing but they do not. If I use chat `to` assertions such as `foo.should.be.a('string');` then that autocompletes.   

Project is TS/Node18.  Curious if ya'll get that kind of autocomplete yourself?

I tried the online [playground](https://stackblitz.com/fork/github/vitest-dev/vitest/tree/main/examples/basic?initialPath=__vitest__) and that works fine. I installed the `vite` and `vitest` packages.  
```
 "typescript": "^4.9.4",
  "vite": "^4.0.4",
  "vitest": "^0.27.3"
```

**Comment by cyberwombat** [comment]
Oh shoot... figured it out.   I had created my own expect methods and created a types file which evidently overwrote the native one.!!!

**Comment by MetRonnie** [comment]
The docs https://vitest.dev/config/globals.html#globals suggests:
```js
// tsconfig.json
{
  "compilerOptions": {
    "types": ["vitest/globals"]
  }
}
```

However I tried this in `jsconfig.json` and couldn't get it to work, instead it was picking up `node_modules\cypress\types\mocha` and `node_modules\@types\chai` in preference to vitest
